docs/podman.1.md

```diff
@@ -94,8 +94,37 @@ has the capability to debug pods/images created by crio.
 | [podman-version(1)](podman-version.1.md)  | Display the Podman version information.                                        |
 | [podman-wait(1)](podman-wait.1.md)        | Wait on one or more containers to stop and print their exit codes.             |
 
+## FILES
+
+
+**libpod.conf** (`/etc/containers/libpod.conf`)
+
+	libpod.conf is the configuration file for all tools using libpod to manage containers
```

Comment:
Libpod.conf is also TOML

Reply:
Removing TOML, users have no idea what that word means.